docs/source/howto/include/scripts/performance_benchmark_base.py

```diff
@@ -28,9 +28,9 @@ def main(code, number, daemon):
     import uuid
 
     from aiida import orm
-    from aiida.common import exceptions
     from aiida.engine import run_get_node, submit
     from aiida.engine.daemon.client import get_daemon_client
+    from aiida.manage import get_manager
```

Comment:
In `verdi` commands I would use the context to get the profile
```python
@click.pass_context
def command(ctx, ...)
    echo.echo(f'Running on profile {ctx.obj.profile.name}')

Reply:
thanks, I tried your suggestion but it turns out `ctx.obj` is empty. I guess this is set somewhere in the `verdi` command group?

Reply:
Ah you are right, my bad. This is a standalone script. Forget this comment then, apologies